torch/fx/proxy.py

```diff
@@ -142,6 +142,14 @@ class TracerBase:
     # Mapping of node name to module scope
     node_name_to_scope: dict[str, tuple[str, type]]
 
+    # Decides whether we represent `NamedTuple` instances into `call_function`
+    # nodes in the IR, or inline them directly as instance arguments to the user
+    # nodes.
+    _proxy_named_tuple: bool
+
+    def __init__(self):
+        self._proxy_named_tuple = True
```

Comment:
@zou3519 not sure what's the best way to do it here, so this is just to get the proof-of-concept working.